lib/TPP/TransformDialectInterpreter.cpp

```diff
@@ -38,9 +38,27 @@ struct TransformDialectInterpreter
   }
 };
 
+struct TransformDropSchedulePass
```

Comment:
I'm surprised this isn't done automatically by the pass, or that there isn't this functionality in MLIR proper...

Reply:
It is not exposed, as Transform is still in test-mode. Something similar is done in IREE too.